src/domain/entities/file.ts

```diff
@@ -1,13 +1,16 @@
-import type { Note } from './note.js';
+import type { NoteInternalId } from './note.js';
 import type User from './user.js';
+import type { Buffer } from 'buffer';
 
 /**
  * File types for storing in object storage
  */
 export enum FileTypes {
   /**
-   * @todo define types
+   * @todo define real types
    */
+  avatar = 'avatar',
+  noteInternalFile = 'noteInternalFile',
```

Comment:
> noteInternalFile
this name is unclear. I'd suggest removing all these mock types and add a `test` type for testing reasons. Real types will be added during future implementation

Reply:
Made only `test` for now